script/Ciel/Day 5/05_01_CIEL05_1DASH.txt

```diff
@@ -253,7 +253,7 @@ After taking a deep breath to calm myself down,
 [sha:daac6f75a83e3477b2860fb4120ece9b620ea8e2]{
 -- Page 39, Offset 10968.
 -- 八階フロアの中は一面の闇だった。
-The entirety of the eighth floor is pitch black.
+The entirety of the eighth floor is pitch-black.
```

Comment:
This is another one that doesn't feel eight with the dash. If I had to guess it's the same diff between
> The ocean is a deep blue
> The deep-blue ocean

but you don't need the article in this case? 
> The night is pitch black
> The pitch-black of night

Reply:
I'll doublecheck